sqlmesh/core/model/common.py

```diff
@@ -137,7 +137,7 @@ def depends_on(cls: t.Type, v: t.Any, values: t.Dict[str, t.Any]) -> t.Optional[
 
 
 properties_validator = field_validator(
-    "table_properties_",
+    "physical_properties_",
```

Comment:
did you need to add view_properties here?

Reply:
I guess I should put all properties isn't it?
`table_properties` and `virtual_properties` as well

Reply:
i'd like to remove table_properties from the model, and only have table_properties be accepted as an input from legacy projects if that makes sense, so internally, we'd only ever have to deal with the two.